prusti-interface/src/environment/mod.rs

```diff
@@ -115,7 +115,8 @@ impl<'tcx> Environment<'tcx> {
         sp: S,
         msg: &str,
         help: &Option<String>,
-        note: &Option<(String, S)>
+        note: &Option<(String, S)>,
+        additional_notes: &Vec<(String, Option<S>)>,
```

Comment:
Could we keep just the vector and call it `notes`? The compiler receives the same thing for both of these anyway.